src/fetchApi.js

```diff
@@ -0,0 +1,14 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/llFawVoYD7zLAfvY9jQn/scores/';
+
+const postApi = (score) => {
+  const scoreResult = fetch(apiUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(score),
+  });
+  return scoreResult.json;
```

Comment:
- [ ] As pointed out earlier, please use `async` and `await`

Reply:
@AnselemOdims okay let me rewrite the logic